chapter_02_repository.asciidoc

```diff
@@ -627,12 +627,14 @@ def test_repository_can_retrieve_a_batch_with_allocations(session):
 //TODO (DS): Picking a descriptive SKU (e.g. 'comfy-sofa') would make this a
 //bit more fun to read.
 // Worth explaining why we have to do a follow up query to get the id inserted?j
-// Why the underscore in _allocations here?
+// Why the underscore in _allocations here? It was already private in the definition
+// of the Batch class in chapter 1. Maybe for consistency we want to make them all
```

Comment:
I'd drop the "Maybe"

Reply:
ok now you're making suggestions for the grammar in _comments_

Reply:
not even comments i wrote!  that's a comment from a tech reviewer :rofl: